src/tests/JIT/opt/perf/doublenegate/doublenegate.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <PropertyGroup>
+    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
+    <DebugType>None</DebugType>
+    <NoStandardLib>True</NoStandardLib>
+    <Noconfig>True</Noconfig>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <GCStressIncompatible>true</GCStressIncompatible>
+  </PropertyGroup>
```

Comment:
You can just delete these lines -- then might as well move the `<DebugType>` up to the first property group.
```
    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
    <NoStandardLib>True</NoStandardLib>
    <Noconfig>True</Noconfig>
    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
    <GCStressIncompatible>true</GCStressIncompatible>
```

Reply:
Sure thing, will do.